src/OrchardCore.Modules/OrchardCore.Flows/Controllers/AdminController.cs

```diff
@@ -111,7 +111,7 @@ private async Task<IEnumerable<ContentTypeDefinition>> GetContainedContentTypesA
         {
             var settings = (await _contentDefinitionManager.GetTypeDefinitionAsync(contentType))?.Parts.SingleOrDefault(x => x.Name == partName)?.GetSettings<FlowPartSettings>();
 
-            if (settings?.ContainedContentTypes?.Length == 0)
+            if (settings?.ContainedContentTypes == null || settings.ContainedContentTypes.Length == 0)
```

Comment:
FYI I checked if the more succinct `items?.Any() == false` pattern would work. It wouldn't, as it (I don't understand why) produces `null`; and it also violates [CA1860](https://learn.microsoft.com/en-us/dotnet/fundamentals/code-analysis/quality-rules/ca1860), nudging you towards `items?.Count == 0`. So, that's not an option.